cmd/virt-handler/Dockerfile

```diff
@@ -25,4 +25,10 @@ RUN dnf -y install libvirt-client \
 
 COPY virt-handler /virt-handler
 
+RUN groupadd -g 1000 libvirt && useradd -u 1000 -g 1000 -N --create-home -s /bin/bash virt-handler && \
+    chown virt-handler /virt-handler
+USER virt-handler
+WORKDIR /home/virt-handler
```

Comment:
Would something like this make any more sense?  I'm not sure I understand why the WORKDIR is set to the virt-handler home directory.

```
COPY virt-handler /virt-bin/virt-handler
WORKDIR /virt-bin
RUN groupadd -g 1000 libvirt && useradd -u 1000 -g 1000 -N virt-handler && chown -R virt-handler /virt-bin
USER virt-handler
```

Reply:
I believe in this case we will need to change `ENTRYPOINT` under `Dcokerfile`, I prefer to leave virt-handler on the old place. I can omit creating of the home directory and remove `WORKDIR`.